Tests/test_pairwise_aligner.py

```diff
@@ -18537,6 +18537,8 @@ def test_pickle_aligner_match_mismatch(self):
             pickled_aligner.extend_right_deletion_score,
         )
         self.assertEqual(aligner.mode, pickled_aligner.mode)
```

Comment:
Above (around line 18485), can you set `aligner.epsilon` to something different than its default value (which is 1.e-6)? Then we can be sure that the pickled value is correct, and not the default value.